draft-ietf-scitt-architecture.md

```diff
@@ -459,19 +458,21 @@ A Transparent Statement remains a valid Signed Statement, and may be registered
 
 Verifiable Data Structure:
 
-: a data structure which supports one or more proof types, such as "inclusion proofs" or "consistency proofs" (as defined in {{-COMETRE}}).
-
+: a data structure which supports one or more proof types, such as "inclusion proofs" or "consistency proofs", for Signed Statements as they are Registered to a Transparency Service.
+SCITT supports multiple Verifiable Data Structures and Receipt formats as defined in {{-COMETRE}}, accommodating different Transparency Service implementations.
+May be referenced as an Append-only log, indicating its immutable nature.
```

Comment:
```suggestion
```

The log is the sequence of full entries, the VDS captures their digest and can be used to emit VDPs, but is not the same thing as the log itself.

Reply:
Yeah this one change I think captures my earlier point. A VDS proves append only, the storage can not do this. So it is reasonable to say "the vds assures an append only log", and maybe also shorthand that as "referenced as an  Append-only log"

Reply:
Leaving this open for discussion as I'm not hearing convergence to remove, or alter the line.

Reply:
My preference would be to drop the line, and clearly distinguish the "Register Sequence of Signed Statements" (*) and the "VDS" elsewhere too. (*) I am not hung up on the particularly name, but something that describes the Signed Statements themselves in the order in which they are committed to the VDS.